homeassistant/components/humidifier/__init__.py

```diff
@@ -169,6 +172,9 @@ def state_attributes(self) -> dict[str, Any]:
         if supported_features & SUPPORT_MODES:
             data[ATTR_MODE] = self.mode
 
+        if supported_features & SUPPORT_OPERATING_STATE:
+            data[ATTR_OPERATING_STATE] = self.operating_state
```

Comment:
ZWave humidifiers expose the current operating state of the humidifier, which is different from the "mode" in which the humidifier is set. For instance, in "auto" mode, the operating state will go back and forth between `Humidifying` and `Idle`. I added this new attribute since I think this exposes useful information that had no good way of being exposed before. My intention is to follow up with UX changes to display this attribute in the Humidifier card when available, just like the Thermostat card does.

Reply:
Changes to the humidifier platform need to be a separate PR. You can either mark this one as draft and wait until the platform changes are merged and rebase, or you can remove the support for this feature in this PR and submit it as a second one, your call.

Reply:
Opened: https://github.com/home-assistant/core/pull/65855

Reply:
I suggest removing all features from this PR that we won't accept immediately to not block this PR.